cli/command/cluster/connectivity.go

```diff
@@ -0,0 +1,53 @@
+package cluster
+
+import (
+	"github.com/dnephin/cobra"
+
+	"github.com/storageos/go-cli/cli"
+	"github.com/storageos/go-cli/cli/command"
+	"github.com/storageos/go-cli/cli/command/formatter"
+)
+
+type connectivityOptions struct {
+	quiet  bool
+	format string
+}
+
+func newConnectivityCommand(storageosCli *command.StorageOSCli) *cobra.Command {
+	var opt connectivityOptions
+
+	cmd := &cobra.Command{
+		Use:   "connectivity [OPTIONS]",
+		Short: "Display connectivity diagnostics for the cluster",
+		Args:  cli.RequiresMinArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runConnectivity(storageosCli, opt)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVarP(&opt.quiet, "quiet", "q", false, "Only display node names")
```

Comment:
The description is a little off? I was expecting it to just dump node names, but it prints the status of each node too.